pkg/mongokit/mongokit_test.go

```diff
@@ -0,0 +1,44 @@
+package mongokit
+
+import (
+	"context"
+
+	"github.com/NTHU-LSALAB/NTHU-Distributed-System/pkg/logkit"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Mongokit", func() {
+	Describe("NewMongoClient", func() {
+		var (
+			mongoClient *MongoClient
+			ctx         context.Context
+			mongoConfig *MongoConfig
+		)
+
+		BeforeEach(func() {
+			ctx = logkit.NewLogger(&logkit.LoggerConfig{
+				Development: true,
+			}).WithContext(context.Background())
+
+			mongoConfig = &MongoConfig{
+				URL:      "mongodb://mongo:27017",
+				Database: "video",
```

Comment:
```suggestion
				Database: "nthu_distributed_system",
```

是 database name 不是 collection name～